### ISSUE TITLE
Should we test APE examples with CI?

### ISSUE THREAD

**Issue by pllim** [astropy/astropy-APEs#50]
As mentioned by @taldcroft in https://github.com/astropy/astropy-APEs/issues/48#issuecomment-446697492 , "But the bigger question of testing APE6 examples is quite good."

**Comment by bsipocz** [comment]
Not all APE API examples are the actual API we end up with, so presumably you don't suggest to test it in the APE repo?

Otherwise, if we don't yet do it, it would be nice to test those examples that actually ended up implemented and working in the initial implementation of the APE.

**Comment by pllim** [comment]
Yeah, I don't know what is the best way to go about this. Maybe something along the line of tutorials where the accepted examples can be in notebooks and tested?
